### DISCUSSION TITLE
Is Winmerge program more recommended in 32 bit or 64 bit version?

### DISCUSSION THREAD

**Discussion by MichelMerlin** [WinMerge/winmerge#2193] [Q&A]
In Many cases, like Notepad++, I have reverted from 64bit to 32bit after experience over the years. Oppositely to the case of OSes, switching APPS from 32bit to 64bit seems to make not much difference in size and speed, yet often causes more losses than gains in compatibility (for plugins, OS, data). What about the case of Winmerge?
Versailles, Sun 28 Jan 2024 22:31:40 +0100

**Comment by sdottaka** [comment]
Basically, the 64-bit version is recommended.
The following functions cannot be used in the 32-bit version.

- Drawing colored emoji using DirectWrite (Options dialog window → Editor category → Rendering Mode)
- Ability to compare PDF files and SVG files as images
- Display additional properties (image size, etc.) in folder comparisons (Display Columns dialog window)
- Compare the latest clipboard contents with the previous clipboard (File → Compare Clipboard menu item)

Additionally, if you install the 64-bit version using the installer, WinMerge32BitPluginProxy.exe will be installed, and 32-bit plug-ins can also be used via this executable file.
